src/api/fileops/fileset.ts

```diff
@@ -47,4 +63,44 @@ export class Fileset<FormDataType extends IFormData<F>, T, D, F> implements IRes
    * public update(data: T): UpdateQuery<T> {}
    * public delete(): DeleteQuery<D> {}
    */
+
+  /**
+   * Subscribe for the RTC records
+   * update file record with a status
+   * @param uploadingProcess {Observable<FilesetInterface<T>>} Observable of the uploading files process
```

Comment:
the `type` comes comes before the param name, like:
```js
* @param {Observable<FilesetInterface<T>>} uploadingProcess Observable of the uploading files process

Reply:
nevermind, the doc will be generated based on the type defined in typescript :+1: